scripts/workers/code_upload_worker_utils/persistent_volume_claim.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: efs-claim
+spec:
+  accessModes:
+    - ReadWriteMany
+  storageClassName: efs-sc
+  resources:
+    requests:
+      storage: 5Gi
```

Comment:
Also, please change this storage to 500 GB

Reply:
The size mentioned here won't affect how much data you could store on EFS. As EFS doesn't have any upper limit we don't have to worry about it. We have to mention some size here because kubernetes needs it. We can keep this value as it is